### PR TITLE
Label Improvements

### PR DESCRIPTION
**Pull request by Jade-Harleyy**
image

### CODE REVIEW

File: Libraries/BarotraumaLibs/BarotraumaCore/Utils/MathUtils.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace Barotrauma
     [Flags]
     public enum Alignment
     {
-        CenterX = 1, Left = 2, Right = 4, CenterY = 8, Top = 16, Bottom = 32,
+        Left = 1, CenterX = 2, Right = 4, Top = 8, CenterY = 16, Bottom = 32,
```
Comment (Regalis11) [inline Libraries/BarotraumaLibs/BarotraumaCore/Utils/MathUtils.cs:13]: Why was this changed? I think it could have some unintended side-effects, such as things previously defaulting to CenterX defaulting to Left, which could cause for example UI layout issues in tons of places.

File: Barotrauma/BarotraumaClient/ClientSource/Items/Components/ItemLabel.cs [hunk 2]
```diff
@@ -60,6 +68,16 @@ public string Text
...
 
+        /*
+        private GUIFont font;
+        [Editable, Serialize("UnscaledSmallFont", IsPropertySaveable.Yes, description: "The label's font.")]
+        public GUIFont Font
+        {
+            get => font;
+            set => font = textBlock.Font = value;
+        }
+        */
```
[the remark this answers is not in the document]
Comment (Jade-Harleyy) [reply Barotrauma/BarotraumaClient/ClientSource/Items/Components/ItemLabel.cs:79]: This and the next were commented out since I couldn't find a way to implement font selection.
I meant to come back to it, but I suppose I had forgot (until now).

File: Libraries/BarotraumaLibs/BarotraumaCore/Utils/MathUtils.cs (hunk 1 above)
Comment (Jade-Harleyy) [reply Libraries/BarotraumaLibs/BarotraumaCore/Utils/MathUtils.cs:13]: I believe I changed this one so the values would appear "in order" (left -> right, top -> bottom), although in hindsight I see this wasn't very necessary and can be reverted.

**Comment by Regalis11** [comment]
This does not seem to be in a reviewable/mergeable state yet (font selection not implemented).
